dist/src/main/java/io/camunda/application/commons/console/ping/PingConsoleRunner.java

```diff
@@ -91,7 +105,8 @@ protected void validateConfiguration() {
       throw new IllegalArgumentException(
           String.format("Ping endpoint %s must be a valid URI.", pingConfiguration.endpoint));
     }
-    if (clusterId == null || clusterId.isBlank()) {
+    if (brokerTopologyManager.getClusterConfiguration().clusterId().isEmpty()
```

Comment:
🙃 It is a bit weird that we use exceptions here but directly catch it again and then log the message. Might make more sense to simply return either, and not create the exceptions/stacktraces 🤔